Activities/Cryptography/UiPath.Cryptography.Activities/DecryptFile.cs

```diff
@@ -61,7 +68,6 @@ public partial class DecryptFile : CodeActivity
         [LocalizedDisplayName(nameof(Resources.Activity_DecryptFile_Property_ContinueOnError_Name))]
         [LocalizedDescription(nameof(Resources.Activity_DecryptFile_Property_ContinueOnError_Description))]
         public InArgument<bool> ContinueOnError { get; set; }
-
```

Comment:
why remove this line? no reason for it

Reply:
Looked like 2 newlines in a row at that moment I guess. I added it back.